modules/geolocation/geolocation.py

```diff
@@ -97,3 +96,37 @@ def gather_point_pairs(self):
             pixelGeoPairs = np.concatenate((pixelGeoPairs, [pair]))
 
         return pixelGeoPairs
+
+    def calculate_pixel_to_geo_mapping(self):
+        """
+        Outputs transform matrix for mapping pixels to geographical points
+
+        Returns
+        -------
+        np.array(shape=(3,3))
+        """
+
+        # Declare 4 matrices
+        # Assign relevant values, shapes and data types
+        A = np.vstack((self.__pixelToGeoPairs[0:3, 0:1].reshape(3, 2).T, [1, 1, 1])).astype(np.float64)
```

Comment:
I think more descriptive names for this would be helpful. For example, call this one firstCoordinateMatrix for A and firstCoordinateVectorSolution for ARHS.

Reply:
sourceCoordinateMatrix or pixelCoordinateMatrix would be better.

Reply:
Also add a comment explaining what all of this slicing and stacking and shaping is doing.